samples/tutorials/gpu/gpu_smoking_check.yaml

```diff
@@ -0,0 +1,70 @@
+"apiVersion": |-
```

Comment:
Do we need to check in this file? We usually do not check in compiled pipelines.

Reply:
removed.